src/modules/shared/Footer/Footer.tsx

```diff
@@ -9,7 +9,7 @@ import top_button_dark
   from '../../../static/buttons/slider-up-button-default_button_dark.svg';
 
 export const Footer: React.FC = () => {
-  const isDarkTheme = false;
+  const isDarkTheme = true;
```

Comment:
Use a global variable from our store that is responsible for the dark theme, so you don't have to make changes later
`const { isDarkTheme  } = useAppSelector(state => state.theme);`
Don't forget to import the `useAppSelector` from `srs/store/hooks`